Presentation/FormatterFactories/Concrete/BookFormatterFactory.cs

```diff
@@ -0,0 +1,31 @@
+using Opcion1LosBorbotones.Domain;
+using Opcion1LosBorbotones.Presentation.Renderer;
+using Opcion1LosBorbotones.Presentation.Renderer.BookFormatter;
+using Opcion1LosBorbotones.Presentation.Renders;
+using Spectre.Console;
+
+namespace Opcion1LosBorbotones.Presentation;
+
+public class BookFormatterFactory : IEntityFormatterFactory<Book>
```

Comment:
The factories are breaking Single Responsibility, because they are creating and rendering the result, they should only create.

You can render the formatted results with the "ResultRenderer" utility like this:
for lists:
ResultRenderer.RenderResults(results);

for single result:
ResultRenderer.RenderResult(result);

Reply:
Done